pkg/controllermanager/controller/seed/lifecycle/reconciler.go

```diff
@@ -132,8 +132,15 @@ func (r *Reconciler) Reconcile(ctx context.Context, req reconcile.Request) (reco
 		return reconcile.Result{RequeueAfter: r.Config.SyncPeriod.Duration}, nil
 	}
 
+	var gardenletOfflineSince interface{}
+	if conditionGardenletReady != nil {
+		gardenletOfflineSince = conditionGardenletReady.LastTransitionTime.UTC()
+	} else {
+		gardenletOfflineSince = "Unknown"
+	}
```

Comment:
I think it can be even further simplified

```suggestion
	gardenletOfflineSince := "Unknown"
	if conditionGardenletReady != nil {
		gardenletOfflineSince = conditionGardenletReady.LastTransitionTime.UTC().String()
	}
```

Reply:
It can't be done, once the variable is initialized its type is set, and it can't be changed.

Reply:
Hm, in `gardenletOfflineSince := "Unknown"`, `gardenletOfflineSince` is string. `conditionGardenletReady.LastTransitionTime.UTC().String()` also evaluates to string. We should not be changing the type of the `gardenletOfflineSince` var.

Reply:
+1 the logger has specific "stringify" logic depending on the underlying type.

Reply:
@timuthy can you elaborate more?

Reply:
Do you mean that the logger has internal "stringify" logic for `time.Time` and we should rather avoid passing already stringified `time.Time`?

Reply:
We could keep the var of type `interface{}` then:
```golang
	var gardenletOfflineSince interface{} = "Unknown"
	if conditionGardenletReady != nil {
		gardenletOfflineSince = conditionGardenletReady.LastTransitionTime.UTC()
	}
```

Is this approach ok for you?

Reply:
@acumino can you revert the latest changes? As said above by Ismail, we should keep the type time.Time.

Reply:
Update the PR.